Userland/Libraries/LibWeb/Fetch/Infrastructure/HTTP/Bodies.cpp

```diff
@@ -13,28 +13,43 @@
 
 namespace Web::Fetch::Infrastructure {
 
-Body::Body(JS::Handle<Streams::ReadableStream> stream)
+JS::NonnullGCPtr<Body> Body::create(JS::VM& vm, JS::NonnullGCPtr<Streams::ReadableStream> stream)
+{
+    return vm.heap().allocate_without_realm<Body>(stream);
+}
+
+JS::NonnullGCPtr<Body> Body::create(JS::VM& vm, JS::NonnullGCPtr<Streams::ReadableStream> stream, SourceType source, Optional<u64> length)
+{
+    return vm.heap().allocate_without_realm<Body>(stream, source, length);
+}
+
+Body::Body(JS::NonnullGCPtr<Streams::ReadableStream> stream)
     : m_stream(move(stream))
 {
 }
 
-Body::Body(JS::Handle<Streams::ReadableStream> stream, SourceType source, Optional<u64> length)
+Body::Body(JS::NonnullGCPtr<Streams::ReadableStream> stream, SourceType source, Optional<u64> length)
     : m_stream(move(stream))
     , m_source(move(source))
     , m_length(move(length))
 {
 }
 
+void Body::visit_edges(JS::Cell::Visitor& visitor)
```

Comment:
`We usually just write `Body::visit_edges(Visitor& visitor)`, no need for all the extra qualifiers.